docs/source/zero-to-binderhub/setup-binderhub.rst

```diff
@@ -452,20 +452,20 @@ clone a repo.
 Use Docker-inside-Docker (DinD)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, BinderHub will build pods with the host Docker installation.
+By default, BinderHub will build images with the host Docker installation.
 This often means you are stuck with whatever version of Docker provided by your
 cloud provider. BinderHub supports an alternative that uses `Docker-in-Docker
 (DinD) <https://hub.docker.com/_/docker>`_. To turn `dind` on, you'll need to set
 the following configuration in your ``config.yaml`` file::
 
+    containerBuilderPod: dind
```

Comment:
In the paragraph above, it sais "binderhub will build images", but we speak about a containerBuilderPod. Isn't this an imageBuilderPod?

I think of images as static artifacts that can be built, and when built they can be used to start a container.

I also want to highlight that maybe `pod` isn't what makes sense here either. In the schema description you write `Selected builder pod type`. It doesn't seem critical to declare that it must be a k8s Pod either, it must just be some `imageBuilderType` I'd say, and if that type leads to creating a DaemonSet with one pod on each node etc, or building the image locally where the binderhub software runs, then so be it.

---

Thinking of that, what happens when `imageBuilderType=disabled` is configured or `dind.enabled=False` as it was before? Is `disabled` really what we should say here, or is it something else like `local` etc? I'm not confident what it means to not have `dind.enabled=False` currently - Would binderhub would still work in some way, and if so, how - and how would we describe that imageBuildingType?

I think I'd suggest `imageBuilderMode` or `imageBuilderType` or similar, but the words container and pod doesn't seem right here.

Reply:
Strictly speaking the build pod shouldn't even need to use a socket, it should be possible to use a tcp connection to the docker/podman host. I managed to get it working a while back but had some other problems, but I think it's worth revisiting in the future as it takes us one step closer to having a K8s BinderHub that's not dependent on the host configuration.

Reply:
If memory serves well, in the context of Podman, I had it working with a tcp socket using repo2podman and a k8s service.
So definitively something to check again.